# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1692356735 -10800
# Parent  6b1bab44ad852183aa679fc3587a2ffc0a4e57e7
Remove CSS form bug 1846611 since it doesn't work. Add additional hunk for bug 1848834.

diff --git a/mail/themes/shared/mail/about3Pane.css b/mail/themes/shared/mail/about3Pane.css
--- a/mail/themes/shared/mail/about3Pane.css
+++ b/mail/themes/shared/mail/about3Pane.css
@@ -592,16 +592,17 @@ menuitem.server[data-server-type="rss"] 
   background-color: var(--folderpane-unread-new-count-background);
 }
 
 #folderTree:focus-within li.selected > .container > .unread-count {
   background-color: var(--folderpane-unread-count-text);
   color: var(--treeitem-background-active);
 }
 
+/* Not working, bug 1846611
 .unread.children.collapsed > .container > .unread-count {
   color: var(--folderpane-unread-count-background);
   background-color: transparent;
   border: 1px solid var(--folderpane-unread-count-background);
   padding-block: 0;
 }
 
 .new-messages.unread.children.collapsed > .container > .unread-count {
@@ -609,16 +610,17 @@ menuitem.server[data-server-type="rss"] 
   border-color: var(--folderpane-unread-new-count-background);
 }
 
 #folderTree:focus-within li.selected.unread.children.collapsed > .container > .unread-count {
   color: var(--folderpane-unread-count-text);
   background-color: transparent;
   border-color: var(--folderpane-unread-count-text);
 }
+*/
 
 .folder-size {
   font-size: .8rem;
   font-weight: bold;
   color: var(--layout-color-3);
   margin-inline-end: 6px;
 }
 
diff --git a/mail/themes/shared/mail/threadPane.css b/mail/themes/shared/mail/threadPane.css
--- a/mail/themes/shared/mail/threadPane.css
+++ b/mail/themes/shared/mail/threadPane.css
@@ -125,19 +125,26 @@
   text-align: end;
 }
 
 /* Add on the end aligned columns a padding when they are at the end of the row. */
 .sizecol-column[colspan="2"],
 .unreadcol-column[colspan="2"],
 .totalcol-column[colspan="2"] {
   /* 25px is the width of the column picker. */
-  padding-inline-end: calc(25px + env(scrollbar-inline-size));
+  padding-inline-end: 25px;
 }
 
+@media (-moz-overlay-scrollbars) {
+  .sizecol-column[colspan="2"],
+  .unreadcol-column[colspan="2"],
+  .totalcol-column[colspan="2"] {
+    padding-inline-end: calc(25px + env(scrollbar-inline-size));
+  }
+}
 
 [is="tree-view-table-body"]:focus > .selected button,
 [is="tree-view-table-body"]:focus-within > .selected button,
 [is="tree-view-table-body"] > .selected:focus-within button {
   color: currentColor;
 }
 
 #threadTree button:not(.twisty, .button-column-picker),
